Modules/IO/ImageBase/include/itkImageSeriesReader.hxx

```diff
@@ -439,9 +439,6 @@ ImageSeriesReader<TOutputImage>::GenerateData()
         {
           nonUniformSampling = true;
           spacingDeviation = Math::abs(outputSpacing[this->m_NumberOfDimensionsInImage] - dirNnorm);
-          itkWarningMacro(<< "Non uniform sampling or missing slices detected , expected " << std::setprecision(14)
-                          << outputSpacing[this->m_NumberOfDimensionsInImage] << " got: " << dirNnorm
-                          << " Deviation of:" << spacingDeviation);
 
           needToUpdateMetaDataDictionaryArray = true;
```

Comment:
metadata field "ITK_non_uniform_sampling_deviation" should only be updated once, at the end

Reply:
ImageSeriesReader doesn't work for 4D or 5D images.

Reply:
I'm pretty sure it works to create a N+1D image from ND images.... Maybe not for DICOM but other formats.

Reply:
I tried with fMRI series - it doesn't.